src/annotator/sidebar.js

```diff
@@ -16,6 +16,12 @@ import { ToolbarController } from './toolbar';
  * @prop {number} height
  */
 
+/**
+ * @typedef {Window|HTMLElement} WindowOrHTMLElement
+ * @typedef {Parameters<WindowOrHTMLElement["addEventListener"]>} ArgAddEventListener
+ * @typedef {{object: WindowOrHTMLElement, eventType: ArgAddEventListener[0], listener:ArgAddEventListener[1]}} RegisteredListener
```

Comment:
Would it be possible to use the more expansive syntax here when there are multiple properties (i.e. multi-line)? It's a little hard for me to read this.

Reply:
I have modified the format, hopefully it is more readeable.